docs/pt/docs/how-to/graphql.md

```diff
@@ -0,0 +1,62 @@
+# GraphQL
+
+Como **FastAPI** é baseado no padrão **ASGI**, é muito fácil integrar qualquer biblioteca **GraphQL** também compatível com ASGI.
```

Comment:
I would use *o* before FastAPI:

"Como **o** FastAPI é baseado..."

and:

...GraphQL que também **é** compatível com **o** ASGI